README.md

```diff
@@ -1,15 +1,30 @@
-[![Build Status](https://travis-ci.org/microservices-demo/catalogue.svg?branch=master)](https://travis-ci.org/microservices-demo/catalogue) [![Coverage Status](https://coveralls.io/repos/github/microservices-demo/catalogue/badge.svg?branch=master)](https://coveralls.io/github/microservices-demo/catalogue?branch=master)
+[![Build Status](https://travis-ci.org/microservices-demo/catalogue.svg?branch=master)](https://travis-ci.org/microservices-demo/catalogue) 
+[![Coverage Status](https://coveralls.io/repos/github/microservices-demo/catalogue/badge.svg?branch=master)](https://coveralls.io/github/microservices-demo/catalogue?branch=master)
+[![Go Report Card](https://goreportcard.com/badge/github.com/microservices-demo/catalogue)](https://goreportcard.com/report/github.com/microservices-demo/catalogue)
 
-# catalogue
-A microservices-demo service that provides catalogue/product information.
+# Catalogue
+A microservices-demo service that provides catalogue/product information. 
+This service is built, tested and released by travis.
 
-This build is built, tested and released by travis.
+## Bugs, Feature Requests and Contributing
+We'd love to see community contributions. We like to keep it simple and use Github issues to track bugs and feature requests and pull requests to manage contributions.
 
-# Test
-`./test/test.sh < python testing file >`. For example: `./test/test.sh unit.py`
+### To build this service
+`docker-compose build`
 
-# Build
-`GROUP=weaveworksdemos COMMIT=test ./scripts/build.sh`
+### To run the service on port 8080
+`docker-compose up`
 
```

Comment:
Add a build heading with subheadings for building with go tools and one with building with Docker